tests/framework/extensions/provisioning/ssh.go

```diff
@@ -29,15 +35,46 @@ func CallSSHTestByName(testName string, node *nodes.Node) error {
 			return err
 		}
 		str_output := output[:strings.IndexByte(output, '\n')]
-		logrus.Info("CheckCPU test on node " + node.PublicIPAddress + " | Cluster agent cpu usage is: " + str_output + "%")
+		logrus.Infof("CheckCPU test on node %s | Current cluster agent cpu usage is: %s%%", node.PublicIPAddress, str_output)
 
 		output_int, err := strconv.ParseFloat(strings.TrimSpace(str_output), 32)
 		if output_int > cpuUsageVar {
-			logrus.Warn("Cluster agent cpu usage is too high on node" + node.PublicIPAddress + " | Current cpu usage is: " + str_output + "%")
+			logrus.Warnf("Cluster agent cpu usage is too high on node %s | Current cluster agent cpu usage is: %s%%", node.PublicIPAddress, str_output)
 		}
 		if err != nil {
 			return err
 		}
+	case nodeReboot:
+		logrus.Infof("Running NodeReboot test on node %s", node.PublicIPAddress)
+		command := "sudo reboot"
+		_, err := node.ExecuteCommand(command)
+		if err != nil {
+			// There is going to be an error because there will be no response in the ExecuteCommand func
+			// The error will always be "wait: remote command exited without exit status or exit signal"
+			logrus.Info(err)
+		}
```

Comment:
```suggestion
		if err != nil && !errors.Is(err, ssh.ExitMissingError) {
			// There is going to be an error because there will be no response in the ExecuteCommand func
			// The error will always be "wait: remote command exited without exit status or exit signal"
			return err
		}
```
https://pkg.go.dev/golang.org/x/crypto/ssh#ExitMissingError